src/old/lib/components/FilterItems/PostDirectionItem.tsx

```diff
@@ -10,6 +10,8 @@ export interface IPostDirectionItemProps {
 
 export const PostDirectionItem: React.FC<IPostDirectionItemProps> = (props) => {
   const { labelName, handleClick, isDisabledFilter, checked } = props;
+  const theme = useTheme();
+  const matches = useMediaQuery(theme.breakpoints.down('sm'));
```

Comment:
Please rename this variable to something more descriptive. E.g. `isMobile` or something similar

Reply:
I have changed the way I get the screen size